veracruz-client/src/tests.rs

```diff
@@ -190,210 +141,12 @@ fn iterate_over_policy(
-                .0,
-            MOCK_ATTESTATION_ENCLAVE_NAME.to_string(),
-        ))
-    });
-
-    iterate_over_policy(policy_directory().as_path(), |policy| {
-        assert!(policy.is_ok());
-        let policy = policy.unwrap();
-        assert!(VeracruzClient::new(
-            trust_path(CLIENT_CERT_FILENAME),
-            trust_path(CLIENT_KEY_FILENAME),
-            &policy
-        )
-        .is_ok());
-    });
-}
-
-#[test]
-/// This function tests loading invalid policy.
-/// Invalid or out-of-time certificate, and invalid or out-of-time enclave cert-time
-fn test_veracruz_client_new_fail() {
-    // set up the attestation result as a mock object
-    let handler = crate::attestation::MockAttestation::attestation_context();
-    handler.expect().returning(|_, _| {
-        Ok((
-            VeracruzClient::pub_read_cert(MOCK_ATTESTATION_ENCLAVE_CERT_FILENAME)
-                .unwrap()
-                .0,
-            MOCK_ATTESTATION_ENCLAVE_NAME.to_string(),
-        ))
-    });
-
-    iterate_over_policy(policy_path("invalid_policy").as_path(), |policy| {
-        if let Ok(policy) = policy {
-            assert!(
-                VeracruzClient::new(
-                    trust_path(CLIENT_CERT_FILENAME),
-                    trust_path(CLIENT_KEY_FILENAME),
-                    &policy
-                )
-                .is_err(),
-                "{:?}",
-                policy
-            );
-        }
-    });
-}
-
-#[test]
-fn test_veracruz_client_new_unmatched_client_certificate() {
-    // set up the attestation result as a mock object
-    let handler = crate::attestation::MockAttestation::attestation_context();
-    handler.expect().returning(|_, _| {
-        Ok((
-            VeracruzClient::pub_read_cert(MOCK_ATTESTATION_ENCLAVE_CERT_FILENAME)
-                .unwrap()
-                .0,
-            MOCK_ATTESTATION_ENCLAVE_NAME.to_string(),
-        ))
-    });
-
-    let policy_json = std::fs::read_to_string(policy_path(POLICY_FILENAME)).unwrap();
-
-    let rst = VeracruzClient::new(
-        trust_path(DATA_CLIENT_CERT_FILENAME),
-        trust_path(CLIENT_KEY_FILENAME),
-        &policy_json,
-    );
-    assert!(rst.is_err());
-}
-
-#[test]
-fn test_veracruz_client_new_unmatched_client_key() {
-    // set up the attestation result as a mock object
-    let handler = crate::attestation::MockAttestation::attestation_context();
-    handler.expect().returning(|_, _| {
-        Ok((
-            VeracruzClient::pub_read_cert(MOCK_ATTESTATION_ENCLAVE_CERT_FILENAME)
-                .unwrap()
-                .0,
-            MOCK_ATTESTATION_ENCLAVE_NAME.to_string(),
-        ))
-    });
-
-    let policy_json = std::fs::read_to_string(policy_path(POLICY_FILENAME)).unwrap();
-
-    let rst = VeracruzClient::new(
-        trust_path(CLIENT_CERT_FILENAME),
-        trust_path(DATA_CLIENT_KEY_FILENAME),
-        &policy_json,
-    );
-    assert!(rst.is_err());
-}
-
 #[actix_rt::test]
 #[should_panic]
```

Comment:
This line and the comment following probably need to delete too.

Reply:
I had removed these lines in last commit but I deleted many other functions too. Do I have to revert all those deletions and then just remove the line you suggested ?